contrib/packs/config.yaml

```diff
@@ -1,2 +1,30 @@
 ---
 pack_group: st2packs
+
+repositories:
+  default:
+    protocol: "ssh"
```

Comment:
I'm Going this this format, which is much easier to configure:

`st2incubator:
    repo: "https://github.com/StackStorm/st2contrib.git"
    subtree: true

  st2contrib:
    repo: "https://github.com/StackStorm/st2contrib.git"
    subtree: true
`